src/test/java/net/datafaker/Issue759Test.java

```diff
@@ -29,13 +37,16 @@ public void run() {
     }
 
     public static void fakeSomeData(Faker faker) {
-        String state = faker.address().stateAbbr();
-        String zipCode = faker.address().zipCodeByState(state);
         try {
+            String state = faker.address().stateAbbr();
+            String zipCode = faker.address().zipCodeByState(state);
             String county = faker.address().countyByZipCode(zipCode);
             assertThat(county).isNotEqualTo(zipCode);
-        } catch (RuntimeException expected) {
-            assertThat(expected).hasMessageStartingWith("County is not configured for postcode " + zipCode);
+        } catch (RuntimeException | Error e) {
+            if (!e.getMessage().startsWith("County is not configured for postcode ")) {
```

Comment:
Are we sure there couldn't be exception/error with null message?

Reply:
Good point. Yes, theoretically it can happen.

Reply:
@snuyanzin Fixed.